cross/nzbget-testing/Makefile

```diff
@@ -1,16 +1,16 @@
 PKG_NAME = nzbget-testing
-PKG_VERS = 15.0
+PKG_VERS = 16.0
+PKG_GIT_REV = 1439
 PKG_EXT = tar.gz
-PKG_DOWNLOAD_METHOD = svn
-PKG_SVN_REV = 1286
-PKG_DIST_NAME = trunk
-PKG_DIST_SITE = https://svn.code.sf.net/p/nzbget/code
-PKG_DIST_FILE = $(PKG_NAME)-r$(PKG_SVN_REV).$(PKG_EXT)
-PKG_DIR = $(PKG_NAME)-r*
+PKG_DOWNLOAD_METHOD = git
```

Comment:
Let's use the official download link instead of a git checkout: https://github.com/nzbget/nzbget/archive/v16.0-r1439.tar.gz.
That also means we can add a digests file.